rskj-core/src/main/java/co/rsk/peg/feeperkb/constants/FeePerKbMainNetConstants.java

```diff
@@ -0,0 +1,36 @@
+package co.rsk.peg.feeperkb.constants;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.AddressBasedAuthorizer;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FeePerKbMainNetConstants extends FeePerKbConstants {
+
+    private static final FeePerKbMainNetConstants instance = new FeePerKbMainNetConstants();
+
+    public FeePerKbMainNetConstants() {
```

Comment:
```suggestion
   private FeePerKbMainNetConstants() {
```
Looks like this class (as well as the original BridgeMainnetConstants` is implemeting a singleton pattern